env0/resource_configuration_variable.go

```diff
@@ -121,6 +138,9 @@ func resourceConfigurationVariableCreate(ctx context.Context, d *schema.Resource
 	isSensitive := d.Get("is_sensitive").(bool)
 	typeAsString := d.Get("type").(string)
 	format := client.Format(d.Get("format").(string))
+	isReadOnly := d.Get("is_read_only").(bool)
+	isRequired := d.Get("is_required").(bool)
```

Comment:
tests for create and update?

Reply:
1435c2c4ca02d115f704fe2e1cc1b22d73779df9

Reply:
32fded71f4ff7b022353c4f41ae77949a4b95c56